platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/PlatformData.java

```diff
@@ -528,7 +528,7 @@ public long getMinimumGenerationNonAncient() {
      * @see PlatformState#getInfoString()
      */
     public String getInfoString() {
-        return new TextTable()
+        final String dataTable = new TextTable()
```

Comment:
A few ideas, think about them and we can discuss/iterate if you don't agree.

- if we have a hash and a mnemonic, could we put those rows next to each other?
- Instead of having the labels `XXX hash`(for the hash mnemonic) and `XXX hash (abbreviated)` (for the full hash in hex), could we instead have `XXX mnemonic` (for the hash mnemonic) and `XXX hash` (for the full hash in hex)?